pkg/sql/schemachanger/scplan/plan_test.go

```diff
@@ -124,6 +124,10 @@ func TestPlanDataDriven(t *testing.T) {
 func validatePlan(t *testing.T, plan *scplan.Plan) {
 	stages := plan.Stages
 	for i, stage := range stages {
+		if stage.IsResetPreCommitStage() {
+			// Skip the reset stage.
+			continue
+		}
```

Comment:
what about this stage would fail to validate?

Reply:
In this iteration when re-planing the planner will re-add a reset stage and this will mess with our assertions. I'll add a comment.